### DISCUSSION TITLE
Add wallet to Registry

### DISCUSSION THREAD

**Discussion by hambolu** [WalletConnect/walletconnect-monorepo#802] [Developer Support]
Please how long does it take to be approved after submitting for Registry listing

**Comment by chadyj** [comment]
@hambolu It should only take a few days?
What are you submitting? I'll take a look.

**Reply by hambolu** [reply]
ok thanks

**Reply by SnailMyth** [reply]
Would you pls tell me about how long does it take ?I just submt one Registry.But maye it have some shortage with information.Because I want to do a fast-quick app,and testing the function of walletconnet.

**Reply by SnailMyth** [reply]
@chadyj

**Reply by chadyj** [reply]
Approvals happen within a few days.  If your app hasn't been approved it probably doesn't meet the registry guidelines.
https://walletconnect.com/registry/guidelines

**Reply by SnailMyth** [reply]
@chadyj i have submit a registry what id named with sevenseawallet acording to the guidelines for free day.What information do I need to provide extraly? Thank you for your guidance!

**Reply by chadyj** [reply]
> @chadyj i have submit a registry what id named with sevenseawallet acording to the guidelines for free day.What information do I need to provide extraly? Thank you for your guidance!

Please refer to https://walletconnect.com/registry/guidelines

**Comment by Seifollahy** [comment]
> Please how long does it take to be approved after submitting for Registry listing

**Reply by chadyj** [reply]
For apps that follow the guidelines, it should be done within a day or two.

**Reply by angrywasp** [reply]
I have a wallet that has been "Pending Review" for 2 months

**Reply by chadyj** [reply]
> I have a wallet that has been "Pending Review" for 2 months

What is your wallet name?

Generally, apps that are not approved don't follow the guidelines https://walletconnect.com/registry/guidelines

**Reply by angrywasp** [reply]
Fundamenta Mobile. It follows the guidelines. I just note that it says "Pending Review" constantly. If it has been reviewed and rejected for some reason, it doesn't provide any feedback to make corrections.

**Reply by chadyj** [reply]
Thanks for bringing it to my attention. The listing has been updated.

**Comment by Francisifegwu** [comment]
Hi @chadyj please I accidentally added the wrong universal link to our wallet connect registry application. Kindly advise on how I can fix this, as the issue has already been closed and archived by you. Thanks.

Issue link: https://github.com/WalletConnect/walletconnect-registry/issues/454

The correct Desktop and Mobile Universal Linking: https://ws.leadwallet.io/api/api/v1/wallet-connect/

Thank you for all you do ❤

**Comment by roadmanfong** [comment]
Hi @chadyj 
I Applied a new wallet to registry a week ago on https://cloud.walletconnect.com/app/registry .
The app store and google play should support now. 
Could you take a look? It's in pending review for a while
The wallet called `Qubic Wallet`

**Reply by chadyj** [reply]
All done!

**Reply by roadmanfong** [reply]
Thanks a lot. 
I just applied another change removing the last slash of  `https://wallet.qubic.app/`

**Comment by khoadopamine** [comment]
Hi @chadyj 
I sign up for a wallet app. it's available on AppStore and GooglePlay
Can you review it? It's in pending review for 1 month. The wallet called `Dopamine`

**Reply by chadyj** [reply]
Rejected because the URL's were invalid.
Please refer to https://walletconnect.com/registry/guidelines

**Comment by kumarnagender09** [comment]
Error
Creating registry listing failed: {}

**Reply by chadyj** [reply]
Can you please post a screenshot of the error and other details so I can research?

**Reply by kumarnagender09** [reply]
> Can you please post a screenshot of the error and other details so I can research?

fixed

**Comment by angrywasp** [comment]
Fixed a borked link and now back to pending review for 2 weeks. Is there anyone watching these listings or is the only way to get action to complain on github? Makes me think whatever system is in place to handle these registry listings isn’t working properly

**Reply by chadyj** [reply]
What's your app?

**Reply by angrywasp** [reply]
Fundamenta mobile.

**Reply by chadyj** [reply]
👍

**Reply by angrywasp** [reply]
Much appreciated

**Comment by guo** [comment]
Hello!
We have submitted a wallet for months, it has not been reviewed. 
Wallet name: ioPay. 
There is already a version in the registry. We are submitting new information to update the existing one.
I really appreciate your review.
Thank you!

**Reply by guo** [reply]
@chadyj possible to help here? Thanks!

**Comment by SnailMyth** [comment]
@chadyj Hi,I sorry to  you bother again,But we are so worried for this.
I sign up for a wallet app. it's available on AppStore and GooglePlay
Can you review it? It's in pending review for some days. The wallet called.
